lib/components/image_banner_inner_portrait.dart

```diff
@@ -76,7 +76,10 @@ class ImageBannerInnerPortrait extends HookConsumerWidget {
                           style: Theme.of(context).textTheme.headlineSmall,
                           maxLines: 2,
                         ),
-                        Text(items[index].productionYear.toString(),
+                        Text(
+                            items[index].productionYear.toString() == "null"
```

Comment:
You can just use ...productionYear == null

Reply:
Ok, I will fix this :)
And fix the PR so it can be pushed

Reply:
When changing as you suggested, it checks if the object is null and the condition is always false. 
It need to check for the value (that prints out as null on the screen), hanse the quotes arround it. 
This is the correct way.

Reply:
item[index].productionYear should only have the type int or null. So checking for null should work without using .toString()
I will review it tomorrow morning.

Reply:
OK :)
I've made the change. 
notice that .toString() was there (from develop), now it's as you wanted :)